runtime/doc/gui_x11.txt

```diff
@@ -728,7 +728,7 @@ means your Vim does not contain clipboard support.
 In the case when you are trying to access the "* or "+ register and Vim has
 no clipboard support, you will see this warning:
 
-  Warning: Clipboard register not available. See :h W24~
+  Warning: Clipboard register not available.  See :h W24~
```

Comment:
This doesn't look correct. The actual warning message is:
```c
	msg(_("W24: Clipboard register not available. See :h W24"));
```
Which has a single space.